legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-snowflake/legend-engine-xt-relationalStore-snowflake-execution/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/driver/vendors/snowflake/SnowflakeCommands.java

```diff
@@ -25,6 +27,8 @@
 
 public class SnowflakeCommands extends RelationalDatabaseCommands
```

Comment:
Where is this being used? For Graph Fetch we have a different flow for snowflake now

Reply:
For test data gen